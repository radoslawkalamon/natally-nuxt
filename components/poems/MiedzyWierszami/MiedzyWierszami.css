.miedzy-wierszami {
  border: 1px solid var(--color-betwixt);
  border-collapse: collapse;
  color: var(--color-betwixt);
  font-family: var(--font-family-serif);
  font-size: var(--font-size-footnotes);
  margin: 0;
  padding: 0;
  table-layout: fixed;
  text-align: center;
  width: 100%;
}

.miedzy-wierszami th,
.miedzy-wierszami td {
  border: 1px solid var(--color-betwixt);
  padding: 0.5em;
}

.miedzy-wierszami__first-column-bold td:first-of-type {
  font-weight: bold;
}

.miedzy-wierszami__poetic {
  color: var(--color-base);
  font-size: var(--font-size-base);
}

.miedzy-wierszami__text-align-left {
  text-align: left;
}

.miedzy-wierszami__text-align-center {
  text-align: center;
}

.miedzy-wierszami__text-align-right {
  text-align: right;
}
