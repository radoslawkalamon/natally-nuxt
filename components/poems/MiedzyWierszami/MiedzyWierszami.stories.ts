import type { Meta, Story } from '@storybook/vue'
import PoemsMiedzyWierszami from '@/components/poems/MiedzyWierszami/MiedzyWierszami.vue'
import WrapperTextPoems from '@/components/wrappers/TextPoems/TextPoems.vue'

const meta: Meta = {
  title: 'Poems / MiÄ™dzy Wierszami',
  component: PoemsMiedzyWierszami
}
export default meta

export const Default: Story = () => ({
  components: {
    PoemsMiedzyWierszami,
    WrapperTextPoems
  },
  template: `<div class="__storybook-section__">
    <WrappersTextPoems>
      <PoemsMiedzyWierszami />
    </WrappersTextPoems>
  </div>`
})
