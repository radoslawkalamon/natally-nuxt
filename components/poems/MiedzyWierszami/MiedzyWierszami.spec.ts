import MiedzyWierszami from '@/components/poems/MiedzyWierszami/MiedzyWierszami.vue'
import { shallPassUnitSanityTest } from '@/devtools/jest.common.spec'

describe('Poems / MiÄ™dzy Wierszami', () => {
  describe('Unit', () => {
    shallPassUnitSanityTest({ component: MiedzyWierszami })
  })
})
