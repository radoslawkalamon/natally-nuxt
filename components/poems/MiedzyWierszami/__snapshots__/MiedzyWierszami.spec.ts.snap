// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Poems / Między Wierszami Unit shall pass unit sanity test 1`] = `
"<table data-test=\\"poems-miedzy-wierszami\\" class=\\"miedzy-wierszami\\">
  <tr>
    <th>Rozmiar</th>
    <th>Klatka</th>
    <th>Talia</th>
    <th>Biodra</th>
  </tr>
  <tr class=\\"miedzy-wierszami__first-column-bold\\">
    <td>34</td>
    <td>88</td>
    <td>70</td>
    <td>92</td>
  </tr>
  <tr class=\\"miedzy-wierszami__poetic miedzy-wierszami__text-align-left\\">
    <td colspan=\\"4\\">
      Znaleźliśmy się między wierszami
    </td>
  </tr>
  <tr class=\\"miedzy-wierszami__first-column-bold\\">
    <td>
      36
    </td>
    <td>92</td>
    <td>74</td>
    <td>96</td>
  </tr>
  <tr class=\\"miedzy-wierszami__poetic miedzy-wierszami__text-align-center\\">
    <td colspan=\\"4\\">
      tabeli rozmiarów
    </td>
  </tr>
  <tr class=\\"miedzy-wierszami__first-column-bold\\">
    <td>38</td>
    <td>96</td>
    <td>78</td>
    <td>100</td>
  </tr>
  <tr class=\\"miedzy-wierszami__poetic miedzy-wierszami__text-align-right\\">
    <td colspan=\\"4\\">
      masowo produkowanego świata.
    </td>
  </tr>
  <tr class=\\"miedzy-wierszami__first-column-bold\\">
    <td>40</td>
    <td>100</td>
    <td>82</td>
    <td>104</td>
  </tr>
</table>"
`;
